domains/rpc/components/SendCardProfileSearchRpc.vue

```diff
@@ -0,0 +1,164 @@
+<script setup lang="ts">
+import { isAddress } from 'web3-utils'
+
+import type { SearchProfileResult } from '@lukso/web-components'
+
+const BLUR_DELAY = 100
+
+const { currentNetwork } = storeToRefs(useAppStore())
+const { search } = useAlgoliaSearch<IndexedProfile>(
+  currentNetwork.value.indexName
+)
+const { receiver } = storeToRefs(useSendStore())
+const isSearchingReceiver = ref<boolean>(false)
+const searchTerm = ref<string | Address | undefined>(receiver.value?.address)
+const hasNoResults = ref<boolean>(false)
+const results = ref<SearchProfileResult[]>()
+
+const searchResults = async () => {
+  const searchResults = await searchProfile(searchTerm.value)
+
+  if (!searchResults || searchResults?.length === 0) {
+    hasNoResults.value = true
+    return
+  }
+
+  hasNoResults.value = false
+  results.value = []
+
+  for (const hit of searchResults) {
+    results.value.push({
+      name: hit.LSP3Profile?.name,
+      address: hit?.address,
```

Comment:
Why `hit?.address` and not `hit.address`?

Reply:
There was an old version that would have null hits.

Reply:
Yeah, also it's just safier to add `?.`. So many times there is an error that you call property of `undefined` ;)